### PR TITLE
[WIP]Fixing linting issue for  pkg/task/fetch.go maya/pkg/task/fetch.go #2160

### PR DESCRIPTION
**Pull request by ABHIYANSHU**
Signed-off-by: Abhinav Srivastava <abhiyanshu11@gmail.com>

**It fixes linting issues for fetch.go*:

**maya/pkg/task/fetch.go** *(optional, in `fixes #2160(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ABHIYANSHU** [comment]
maya/pkg/task/fetch.go #703

File: pkg/task/fetch.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package task
```
Comment (princerachit) [inline pkg/task/fetch.go]: Remove this line or add some meaningful comment.

File: pkg/task/fetch.go [hunk 2]
```diff
@@ -47,11 +48,11 @@ type K8sTaskSpecFetcher struct {
 	k8sClient *m_k8s_client.K8sClient
 }
 
-// FetchSpec returns specifications of a provided task in yaml
+// This is an implementation of TaskSpecFetcher interface
 // string format
 //
 // NOTE:
-//  This is an implementation of TaskSpecFetcher interface
+// Fetch returns specifications of a provided task in yaml
```
Comment (princerachit) [inline pkg/task/fetch.go]: Undo these changes and replace `FetchSpec` to `Fetch` in the actual comment.

File: types/profiles/v1/doc.go [hunk 3]
```diff
@@ -20,4 +20,6 @@ limitations under the License.
 // Profiles are meant to be used by storage admins to communicate
 // their volume requirements to Maya without being involved into providing
 // exact values or numbers.
+
+// Package v1
```
Comment (princerachit) [inline types/profiles/v1/doc.go]: Not meaningful.

File: types/v1/labels.go [hunk 4]
```diff
@@ -29,6 +30,7 @@ const (
 // defaults
 type EnvironmentVariableDefaults string
 
+// const is used to define EnvironmentVariableDefaults
```
Comment (princerachit) [inline types/v1/labels.go]: I don't see relevant issue description for this change in your PR description. Please link the issue. This is not required.

File: types/v1/labels.go [hunk 5]
```diff
@@ -177,14 +179,13 @@ const (
...
 )
 
-// OrchProviderProfileLabel is a typed label to determine orchestration provider
-// profile's values.
+// OrchProviderProfileLabel is a typed label to determine orchestration provider profile's values.
 type OrchProviderProfileLabel string
 
 const (
-	// Label / Tag for an orchestrator profile name
+	// OrchProfileNameLb1 is Label / Tag for an orchestrator profile name
```
Comment (princerachit) [inline types/v1/labels.go]: Please link the issue to this pr.
